src/assets/locales/zh-tw.json

```diff
@@ -590,9 +590,6 @@
   "support.supported": "supported",
   "support.unsupported": "unsupported",
   "support.update-browser": "Please try switching or updating to a newer browser",
-  "tips.desktop.invite": "Nobody else is here. Use the invite button in the bottom left to share this room.",
-  "tips.desktop.locomotion": "Use the {moveKeys} keys to move. Hold shift to boost.",
-  "tips.desktop.look": "Welcome to {appName}! Let's take a quick tour. 👋 Click and drag to look around.",
   "tips.desktop.turning": "Perfect. Use the {turnLeftKey} and {turnRightKey} keys to rotate.",
```

Comment:
I missed this in my previous list.  I guess this should either be removed, or have the click and drag instructions added.  Also, now that I'm looking here, why are the invite and locomotion tips removed for desktop, but not mobile?